api-reference/10 UI Widgets/Errors and Warnings/E1035.md

```diff
@@ -6,3 +6,17 @@ id: ErrorsUIWidgets.E1035
 Occurs if the editor cannot be created for a [Form](/api-reference/10%20UI%20Widgets/dxForm '/Documentation/ApiReference/UI_Widgets/dxForm/') item because of an internal error.
 
 ---
+
+This error can appear in the following cases:
+
+- **Invalid editorType value**                
+Refer to the [editorType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorType) description for a list of accepted values.
+
+- **Missing modules**           
+The **Form** automatically imports modules only for basic editors. This helps reduce the bundle size. If you use other editors, import their modules manually. Refer to the [editorType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorType) description for more information.
+
+- **Insufficient empty item configuration**            
+[Empty items](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/) require that the [itemType](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/EmptyItem/#itemType) option is set to *"empty"*.
+
+- **Incorrect editorOptions configuration**         
+[editorOptions](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#editorOptions) configure the editor specified by the **editorType**. Check that **editorOptions** matches the **editorType**. Review the editor's API reference and ensure that all used editor options have correct values.
```

Comment:
`Check that **editorOptions** matches the **editorType**.`

I think it's better to change "Check that" to "Ensure that". Also, is **editorOptions** a property? For example, if it's a property, can we say:

`Ensure that the **editorOptions** property matches the **editorType**.`